client/signup/steps/design-picker/index.jsx

```diff
@@ -58,10 +59,9 @@ export default function DesignPickerStep( props ) {
 	const [ selectedDesign, setSelectedDesign ] = useState( null );
 	const scrollTop = useRef( 0 );
 
-	const isDIFMStoreFlow = 'do-it-for-me-store' === props.flowName;
-
 	const getThemeFilters = () => {
-		if ( props.useDIFMThemes ) {
+		if ( useDIFMThemes ) {
+			const isDIFMStoreFlow = 'do-it-for-me-store' === props.flowName;
```

Comment:
Non-blocking: by deriving the theme filtering slug this way, it might be a signal that we can introduce a theme filtering prop defaulting to `'auto-loading-homepage,full-site-editing'` so it's more extensible. However, it's not that important at the moment so please keep it this way :)

Reply:
Indeed! This is the old design picker, and I am discussing with the team whether we should shift to the new one (p1692006665682789-slack-C028NRBP2AU). So this component might just be removed soon :)

Reply:
TIL. It will definitely be better to utilize the same design picker :)